src/constants/indicators.js

```diff
@@ -219,6 +219,7 @@ export default {
     name: 'Jobs accessible by car within 30 minutes',
     format: FORMATS.PERCENT_ZERO_OR_ONE_DECIMAL,
     contextual: false,
+    hideForWesternSydney: true,
```

Comment:
I would feel a lot better about something like: 

```
hideForCities: ['western-sydney']
```

Reply:
The idea is that all this work will be thrown out in a week because the Western Sydney treatment will be re-done, hence the lack of generalisation.

Reply:
When it's known how we will actually treat 'non cities' then we can come up with a solution to suit.

Reply:
I've added https://govausites.atlassian.net/browse/SC-141 that references removing the `SC-140` code so we have no mess left behind.